docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2234.md

```diff
@@ -0,0 +1,38 @@
+# "Custom" capabilities implementation
+
+Test verifies that admin can specify custom scripts in the platform settings and allow users to select them in the **Run capabilities** menu.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- Test tool is available to the non-admin user
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Open the **PREFERENCES** tab | |
+| 4 | Click the **Launch** tab | |
+| 5 | Find the **`launch.capabilities`** preference | |
+| 6 | Save the preference value, then clear and input the following json <ul> `{` <ul> `"testCapability1": {` <ul> `"description": "Custom test capability 1"`, <br> `"commands": [` <ul> `"echo testLine1"`, <br> `"echo 'testLine1' > ~/testFile1.txt"` </ul> `]` </ul> `},` <br> `"testCapability2": {` <ul> `"description": "Custom test capability 2",` <br> `"commands": [` <ul> `"echo testLine2",` <br> `"echo 'testLine2' > ~/testFile1.txt"` </ul> `]` </ul> `}` </ul> `}` |
```

Comment:
> `"echo 'testLine2' > ~/testFile1.txt"`

Fix to: `"echo 'testLine2' >> ~/testFile1.txt"`

Reply:
Done